### DISCUSSION TITLE
Add Tailwind CSS icon

### DISCUSSION THREAD

**Discussion by Monson2002** [tailwindlabs/heroicons#915] [Ideas]
Can someone add an icon for Tailwind CSS. It would be much helpful.

**Comment by zefir-git** [comment]
https://simpleicons.org has a TailwindCSS icon, as well as thousands of other brand icons.

![image](https://user-images.githubusercontent.com/66572326/221116263-7df77e71-ca68-463d-becd-f11f8c97e949.png)

If you want to colour the icon using CSS `color` or tailwind `text-color-…` you need to add `fill="currentColor"` to the `<svg>` tag.
